pkg/kubelet/apis/config/validation/validation_test.go

```diff
@@ -526,22 +526,21 @@ func TestValidateKubeletConfiguration(t *testing.T) {
 	}, {
 		name: "imageMaximumGCAge should not be specified without feature gate",
 		configure: func(conf *kubeletconfig.KubeletConfiguration) *kubeletconfig.KubeletConfiguration {
+			conf.FeatureGates = map[string]bool{"ImageMaximumGCAge": false}
 			conf.ImageMaximumGCAge = metav1.Duration{Duration: 1}
 			return conf
 		},
 		errMsg: "invalid configuration: ImageMaximumGCAge feature gate is required for Kubelet configuration option ImageMaximumGCAge",
 	}, {
 		name: "imageMaximumGCAge should not be negative",
 		configure: func(conf *kubeletconfig.KubeletConfiguration) *kubeletconfig.KubeletConfiguration {
-			conf.FeatureGates = map[string]bool{"ImageMaximumGCAge": true}
 			conf.ImageMaximumGCAge = metav1.Duration{Duration: -1}
 			return conf
 		},
 		errMsg: "invalid configuration: imageMaximumGCAge -1ns must not be negative",
 	}, {
 		name: "imageMaximumGCAge should not be less than imageMinimumGCAge",
 		configure: func(conf *kubeletconfig.KubeletConfiguration) *kubeletconfig.KubeletConfiguration {
-			conf.FeatureGates = map[string]bool{"ImageMaximumGCAge": true}
```

Comment:
keep these lines... if we have to toggle the gate back off, we don't want to have to mess with this unit test to test this behavior